scripts/zkai_attention_kv_eight_head_air_private_softmax_table_lookup_gate.py

```
@@ -0,0 +1,853 @@
+        "claim_boundary": CLAIM_BOUNDARY,
+        "lookup_status": LOOKUP_STATUS,
+        "fused_component_status": FUSED_COMPONENT_STATUS,
+        "next_backend_step": NEXT_BACKEND_STEP,
+        "timing_policy": TIMING_POLICY,
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+        "single_head_comparison": single_head_comparison(),
+    }
+    for key, expected in expected_scalars.items():
+        if payload.get(key) != expected:
+            raise AttentionKvAirPrivateSoftmaxTableLookupGateError(f"{key} drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)